src/assets/Dispositions.js

```diff
@@ -0,0 +1,7 @@
+const dispositions = [
+    'Good with other animals', 
+    'Good with children', 
+    'Animal must be leased at all times'
```

Comment:
A small typo above - Animal must be leashed at all times but instead it's being leased

Reply:
Thanks for catching. Fixed.